solr/core/src/java/org/apache/solr/cli/SolrCLI.java

```diff
@@ -145,9 +145,16 @@ public class SolrCLI implements CLIO {
                   + getDefaultSolrUrl()
                   + '.')
           .build();
+
   public static final Option OPTION_VERBOSE =
       Option.builder("v")
           .longOpt("verbose")
+          .deprecated(
+              DeprecatedAttributes.builder()
+                  .setForRemoval(true)
+                  .setSince("9.8")
+                  .setDescription("Use --debug instead")
+                  .get())
```

Comment:
@epugh could you review this file again? I think this should be undone if we won't introduce `--debug` and deprecate `-v` for version (below too).